docs/src/main/asciidoc/config-yaml.adoc

```diff
@@ -57,8 +57,8 @@ quarkus:
 # REST Client configuration property
 quarkus:
   rest-client:
-    org.acme.restclient.CountriesService:
-      url: https://restcountries.eu/rest
+    org.acme.rest.client.ExtensionsService:
+      url: https://stage.code.quarkus.io/api
```

Comment:
Hmmm, is it a good idea to use a staging instance?

Reply:
this was requested - https://github.com/quarkusio/quarkus/issues/20810#issuecomment-945759283

Reply:
I can see that :). But I'm not very convinced it's a good idea. @ia3andy don't you think it would be better to use the production API? I don't expect a ton of load and availability is important given you have people from all over the world testing this in different time zones.

Reply:
staging is as available as production @gsmet

Reply:
hmm, I don't see that much of a difference. A matter of preference? Let me know if I should change it.

Reply:
staging is better, it has the same availability, we already have tests using it and most of all at least it doesn't affect our usage statistics

Reply:
OK, I'll keep my "I told you so" for later then ;).

Reply:
Oh that's nasty :)